update on how the defaults work

walrusmcd opened the pull request:
update to add what pixel nomalization is defaulted to, and what color space is defaulted to .

opbld34 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/get-started-uwp?branch=pr-en-us-13) |
[docs/get-started-uwp.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/get-started-uwp.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/get-started-uwp?branch=pr-en-us-13) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C24%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201808242110391119-13%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OLWsoLnmcEFEJ%2FqaVw5A5Ig6%2F9QD93SYuvdxkSbciwQ%3D&st=2018-08-24T21%3A06%3A44Z&se=2018-09-24T21%3A11%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

walrusmcd committed:
update on how the defaults work

update to add what pixel nomalization is defaulted to, and what color space is defaulted to .
